ui/app/AppLayouts/Profile/stores/ProfileStore.qml

```diff
@@ -97,10 +103,15 @@ QtObject {
         root.profileModule.setBio(bio)
     }
 
+    // TODO: remove old API
     function storeProfileShowcasePreferences() {
         root.profileModule.storeProfileShowcasePreferences()
     }
 
+    function saveProfileShowcasePreferences(json) {
+        root.profileModule.saveProfileShowcasePreferences()
```

Comment:
Missing handling the `json` param? This is why I hate exposing the strongly typed backend API via a store... You won't even get a warning here

Reply:
fixed